### PR TITLE
Add the TextField component

### PR DESCRIPTION
**Pull request by illright**

### CODE REVIEW

**Comment by illright** [comment]
I wouldn't want to include the SCSS inline, because the styles get quite extensive and it's easier to separate them. Therefore, I'd like the SCSS files to stay. The `./button/button.svelte` is just for that index file, the rest of the world gets a nice interface.

File: src/index.js [hunk 1]
```diff
@@ -1 +1,3 @@
 export { default as Button } from './button/button.svelte';
+
```
[the remark this answers is not in the document]
Comment (illright) [reply src/index.js:2]: Yeah

File: src/text-field/index.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+import TextField from './text-field.svelte';
+export default TextField;
```
[the remark this answers is not in the document]
Comment (aabounegm) [reply src/text-field/index.js:2]: Also, why is this file even needed when in _src/index.js_ you import the Svelte component directly?

[the remark this answers is not in the document]
Comment (illright) [reply src/text-field/index.js:2]: It was kept as an alternative, though honestly it is not necessary. Will remove

**Pull request merged**